services/brig/src/Brig/Data/Connection.hs

```diff
@@ -27,19 +27,20 @@ import Control.Monad.IO.Class
 import Data.Functor.Identity
 import Data.Id
 import Data.Int
+import Data.Json.Util (toUTCTimeMillis, fromUTCTimeMillis)
 import Data.List (foldl')
 import Data.Range
 import Data.Time (UTCTime, getCurrentTime)
 
 connectUsers :: UserId -> [(UserId, ConvId)] -> AppIO [UserConnection]
 connectUsers from to = do
-    now <- liftIO getCurrentTime
+    now <- toUTCTimeMillis <$> liftIO getCurrentTime
     retry x5 $ batch $ do
         setType BatchLogged
         setConsistency Quorum
         forM_ to $ \(u, c) -> do
-            addPrepQuery connectionInsert (from, u, Accepted, now, Nothing, c)
-            addPrepQuery connectionInsert (u, from, Accepted, now, Nothing, c)
+            addPrepQuery connectionInsert (from, u, Accepted, fromUTCTimeMillis now, Nothing, c)
```

Comment:
Perhaps we could have an instance of `Cql` for `UTCTimeMillis`?

Reply:
https://github.com/wireapp/wire-server/pull/366/commits/bb3c2559843bc3b661c54052395a430f38293e65